src/libraries/Microsoft.VisualBasic.Core/src/Microsoft/VisualBasic/CompilerServices/Symbols.vb

```diff
@@ -916,6 +921,8 @@ Namespace Microsoft.VisualBasic.CompilerServices
             ' For a WinRT object, we want to treat members of it's collection interfaces as members of the object 
             ' itself. So GetMembers calls here to find the member in all the collection interfaces that this object 
             ' implements.
+            <UnconditionalSuppressMessage("ReflectionAnalysis", "IL2065:UnrecognizedReflectionPattern",
+                Justification:="_type is annotated with .All, so it's Interfaces will be annotated as well and it is safe to call GetMember on the Interfaces")>
```

Comment:
We should mark these with the issue, so we can clean it up once the linker is fixed.

[https://github.com/mono/linker/issues/1731](https://github.com/mono/linker/issues/1731)